railties/lib/rails/generators/rails/app/templates/config/deploy.yml.tt

```diff
@@ -31,11 +31,13 @@ env:
   # clear:
   #   DB_HOST: 192.168.0.2
 
+<% unless skip_active_storage? && !sqlite3? -%>
```

Comment:
```suggestion
<% if sqlite3? || !skip_active_storage? %>
```

Reply:
Background for the suggestion above:
- I prefer using `if` when there are multiple conditions.
- Changed the order to make this order as the comment says "for sqlite database files and local Active Storage files.", sqlite comes first then Active Storage
- ERB closing `%>` should not need `-`

Reply:
In PR https://github.com/rails/rails/pull/51837/files I made a `skip_storage?` method. I think it's probably best to reuse or port it in this PR

Reply:
Regarding the first and second points, in PR https://github.com/rails/rails/pull/51837 I made a `skip_storage?` method at the suggestion of the reviewer. I have ported it in this PR, let me know what you think.